lib/scala/runtime-version-manager/src/main/scala/org/enso/runtimeversionmanager/runner/Runner.scala

```diff
@@ -222,7 +222,11 @@ class Runner(
         else Seq()
 
       val command = Seq(javaCommand.executableName) ++
-        jvmArguments ++ loggingConnectionArguments ++ runSettings.runnerArguments
+        Seq(
+          "-XX:ActiveProcessorCount=2",
+          "-Xmx8192m"
```

Comment:
@hubertp Just randomly browsing through recent PRs, I have noticed this change. I assume it has something to do with performance. Could you share how the performance was affected? And give us more insight on why you have chosen these two cmdline options?